portal-impl/src/com/liferay/portal/service.xml

```diff
@@ -289,21 +289,47 @@
 		<reference entity="User" package-path="com.liferay.portal" />
 		<reference entity="Website" package-path="com.liferay.portal" />
 	</entity>
-	<entity local-service="false" name="Country" remote-service="true">
+	<entity local-service="true" name="Country" remote-service="true" uuid="true">
 
 		<!-- PK fields -->
 
 		<column name="countryId" primary="true" type="long" />
 
+		<!-- Audit fields -->
+
+		<column name="companyId" type="long" />
+		<column name="userId" type="long" />
+		<column name="userName" type="String" />
+		<column name="createDate" type="Date" />
+		<column name="modifiedDate" type="Date" />
+
 		<!-- Other fields -->
 
 		<column name="name" type="String" />
+		<column name="billingAllowed" type="boolean" />
+		<column name="shippingAllowed" type="boolean" />
 		<column name="a2" type="String" />
 		<column name="a3" type="String" />
 		<column name="number" type="String" />
+		<column name="subjectToVAT" type="boolean" />
 		<column name="idd" type="String" />
 		<column name="zipRequired" type="boolean" />
+		<column name="position" type="double" />
 		<column name="active" type="boolean" />
+		<column name="groupFilterEnabled" type="boolean" />
+		<column name="lastPublishDate" type="Date" />
+
+		<!-- Localized entity -->
+
+		<localized-entity>
+			<localized-column name="title" />
+
+			<!-- Finder methods -->
+
+			<finder name="Title" return-type="country">
+				<finder-column name="title" />
+			</finder>
```

Comment:
Is this finder needed? I don't see this in the CommerceCountry service.xml. 

It'd be easier if you leave out all the finders for now and add it later when you work on the service layer, so you know what finders you need.